src/ui/components/Transfer.js

```diff
@@ -97,12 +106,37 @@ export const Transfer = ({
       });
       updateLedger(nextLedger);
       setAmount("");
+      setHasUnsavedChanges(!hasUnsavedChanges);
```

Comment:
This still needs changed to `setHasUnsavedChanges(true);`

Remember to test the case where 2 transfers are done before saving to disk.